bazel/platforms.bzl

```diff
@@ -16,3 +16,9 @@ PLATFORM_CPU_RISCV64 = ("@platforms//cpu:riscv64")
 
 
 PLATFORM_OS_MACOS = ("@platforms//os:macos")
+
+PLATFORM_OS_LINUX = ("@platforms//os:linux")
+
+PLATFORM_OS_ANDROID = ("@platforms//os:android")
+
+PLATFORM_OS_FREEBSD = ("@platforms//os:freebsd")
```

Comment:
Note: this may look weird that we have to provide this file and not use `@platforms//...` directly but this comes from the fact that we have internal definitions for these cpu/platforms that are different from upstream Bazel. I hope we'll be able to remove this technical debt one day.